src/Java.Interop/Java.Interop/JniRuntime.JniValueManager.cs

```diff
@@ -386,18 +386,9 @@ static Type GetPeerType ([DynamicallyAccessedMembers (Constructors)] Type type)
 			[return: DynamicallyAccessedMembers (Constructors)]
 			static Type? GetInvokerType (Type type)
 			{
-				const string suffix = "Invoker";
-
 				// https://github.com/xamarin/xamarin-android/blob/5472eec991cc075e4b0c09cd98a2331fb93aa0f3/src/Microsoft.Android.Sdk.ILLink/MarkJavaObjects.cs#L176-L186
-				const string assemblyGetTypeMessage = "'Invoker' types are preserved by the MarkJavaObjects trimmer step.";
 				const string makeGenericTypeMessage = "Generic 'Invoker' types are preserved by the MarkJavaObjects trimmer step.";
 
-				[UnconditionalSuppressMessage ("Trimming", "IL2026", Justification = assemblyGetTypeMessage)]
-				[UnconditionalSuppressMessage ("Trimming", "IL2073", Justification = assemblyGetTypeMessage)]
-				[return: DynamicallyAccessedMembers (Constructors)]
-				static Type? AssemblyGetType (Assembly assembly, string typeName) =>
-					assembly.GetType (typeName);
```

Comment:
It's good to remove this, it's probably not called on Android anyway, and this is used instead:
* https://github.com/dotnet/android/blob/63f1e6c709c4e66601042ac6dcefeefa195c7a8f/src/Mono.Android/Android.Runtime/AndroidRuntime.cs#L612